extensions/hawkbit-extension-artifact-repository-s3/pom.xml

```diff
@@ -0,0 +1,78 @@
+<!--
+
+    Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+   <parent>
+      <groupId>org.eclipse.hawkbit</groupId>
+      <artifactId>hawkbit-extensions-parent</artifactId>
+      <version>0.2.0-SNAPSHOT</version>
+   </parent>
+   <artifactId>hawkbit-extension-artifact-repository-s3</artifactId>
+   <name>hawkBit :: S3 Repository</name>
+
+
+   <dependencies>
+      <dependency>
```

Comment:
We can declare the version for the extensions when we move the extensions to the extra github repository for the extensions. Currently it's within the maven-multimodule and managed by hawkBit.